en/docs/learn/design-api/advanced-topics/control-api-visibility-and-subscription-availability-in-developer-portal.md

```diff
@@ -1,4 +1,4 @@
-# Control API Visibility and Subscription Availability in the Developer Portal
+# Control API Visibility, Subscription Availability and API documentation visibility in the Developer Portal
```

Comment:
This doc can be moved under the API documentation section?

Reply:
Yes we can. I referred APIM 2.6.0 docs when creating this section and there the doc visibility was explained after Subscription availability section.

Reply:
fixed in 9234c7c